### PR TITLE
Network selector

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Resolve #339 #303 #399 #391 #403 

### Steps to test

1. run `make run lacchain`
2. go to test webapp

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/NetworkSelector/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+NetworkSelector.propTypes = {
+  title: PropTypes.string,
+  options: PropTypes.array,
+  networkLogo: PropTypes.string
+}
+
+NetworkSelector.defaultProps = {
+  title: '',
+  options: [
```
Comment (xavier506) [inline webapp/src/components/NetworkSelector/index.js:73]: are we setting these as environmental variables?

We currently don't have a Telos or Wax instance running

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/language/es.lacchain.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
     "/entities>sidebar": "Entidades",
     "/entities>title": "Entities - Panel",
     "/entities>heading": "Entities",
-    "/lacchain/network>sidebar": "LACChain Topologia de rede",
-    "/lacchain/network>title": "LACChain Topologia de rede - Panel",
-    "/lacchain/network>heading": "LACChain Topologia de rede",
+    "/lacchain/network>sidebar": "Topologia de red",
```
Comment (xavier506) [inline webapp/src/language/es.lacchain.json:6]: @tetogomez 

English should read Network topology , please read comment in issue

https://github.com/eoscostarica/eosio-dashboard/issues/403

File: .env.lacchain [hunk 3]
```diff
@@ -71,4 +71,7 @@ REACT_APP_EOS_USE_BP_JSON_ON_CHAIN=false
 REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
-REACT_APP_TOKEN_SYMBOL=
\ No newline at end of file
+REACT_APP_TOKEN_SYMBOL=
+REACT_APP_EOS_MAINNET=https://mainnet.eosio.cr
+REACT_APP_JUNGLE_TESTNET=http://testnet.eosio.cr
```
Comment (xavier506) [inline .env.lacchain:76]: https://

Comment (xavier506) [reply .env.lacchain:76]: we should add these variables to the remaining `.env` files also

Comment (tetogomez) [reply .env.lacchain]: done

**Review by adriexnet — APPROVED** [review]

**Pull request merged**